### PR TITLE
Rasika_Assignment_10

### PR DESCRIPTION
**Pull request by RasikaNitin**
image

### CODE REVIEW

File: src/rasika/Assignment10.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package rasika;
+
+public class Assignment10 {
+	int count = 0;
+	int sum = 0;
+	void verifyPrimeNumber(int number) {
+		boolean status = true;
+		for(int index = 2; index < number; index++) {
+			if(number % 2 == 0) {
```
Comment (surabhi-9) [inline src/rasika/Assignment10.java:9]: Validation is wrong. have to use like number % index = = 0 then booleanFlag=false, then break

File: src/rasika/Assignment10.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package rasika;
+
+public class Assignment10 {
+	int count = 0;
+	int sum = 0;
+	void verifyPrimeNumber(int number) {
+		boolean status = true;
```
Comment (surabhi-9) [inline src/rasika/Assignment10.java:7]: try to use meaningfull variable name like isPrimeNumber .

**Review by surabhi-9 — CHANGES_REQUESTED** [review]
Please rewrite the validation for prime number again with correct logic

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me!
Keep doing well!

**Pull request merged**